packages/extension/app/background.coffee

```diff
@@ -177,19 +172,16 @@ automation = {
     .then(fn)
 
   lastFocusedWindow: ->
```

Comment:
avoid all of the `Promise.try` stuff and use `Promise.method` during the defining of the method such as: `lastFocusedWindow: Promise.method ->`